patches/@wordpress+block-editor+14.11.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
+index aa2e9a5..f2d8481 100644
+--- a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
++++ b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
+@@ -192,6 +192,7 @@ class Inserter extends Component {
+         shift: true
+       },
+       onToggle: this.onToggle,
++      open: this.props.open,
```

Comment:
In Gutenberg core, the block appender's "Browse all" button toggles a global `isInserterOpened` state, which controls the visibility of the block inserter in the sidebar. 

The [Gutenberg Framework](https://wordpress.org/gutenberg-framework/docs/basic-concepts/ui#the-block-inserter) guides to use the `Inserter` component for rendering the block inserter, this differs from the aforementioned sidebar block inserter. Unfortunately, the `Inserter` component has no knowledge of the `isInserterOpened` state. 

Therefore, exposing this `open` prop allows us to control the block inserter's visibility via the global `isInserterOpened` state.